actionpack/lib/action_controller/metal/request_forgery_protection.rb

```diff
@@ -128,6 +132,15 @@ def protect_from_forgery(options = {})
         append_after_action :verify_same_origin_request
       end
 
+      # Turn off request forgery protection. This is a wrapper for
+      #
+      #   skip_before_action :verify_authenticity_token
+      #
+      # See skip_before_action for allowed options.
```

Comment:
Small nitpicks, might use `+skip_before_action+` for [rdoc](https://docs.ruby-lang.org/en/2.1.0/RDoc/Markup.html#class-RDoc::Markup-label-Text+Markup)